src/cortex_cli/cortex_cli.py

```diff
@@ -274,6 +255,8 @@ def init(  # pylint: disable=too-many-arguments
         if pid:
             logger.info('Active token manager (PID %s) will be killed.', pid)
             Process(pid).terminate()
+        # Remove tokens file to start from scratch after init
+        os.remove(tokens_file)
```

Comment:
If file exists, it is removed. Also, if file exists, the that means the user confirmed they want to overwrite it (in the `_validate_path` callback). I see how these two things are in sync, but have a feeling it's a bit too loose. At first I thought the file should be deleted inside of `_validate_path`, but that's semantically wrong. So, I guess, I'm just thinking out loud, and nothing should be changed :)